src/nodeconductor_test/helpers.py

```diff
@@ -70,6 +78,16 @@ def _remove_action(driver):
     remove_field.click()
 
 
+def _go_to_tab(driver, css_selector=None, xpath=None, link_text=None, tries_left=3):
+    print 'Go to "%s" tab' % css_selector
+    WebDriverWait(driver, BaseSettings.tab_visible_time_wait).until(
+        EC.visibility_of_element_located((By.CSS_SELECTOR, css_selector)))
+    force_click(driver, css_selector=css_selector)
+    print '"%s" tab was successfully choosen' % css_selector
+    time.sleep(BaseSettings.click_time_wait)
+    time.sleep(BaseSettings.click_time_wait)
```

Comment:
This function does not support `xpath`, `link_text` and  `tries_left`arguments. Why does it receive them?